lib/index.js

```diff
@@ -98,16 +98,16 @@ function run (compiler, compilation, done) {
 // Below are the methods we use to resolve data from each type. Very
 // straightforward, really.
 function renderData (obj) {
-  return obj.data
+  return transformRaw(obj, obj.data)
```

Comment:
Ah yeah I will go ahead and remove that. I was thinking if you wanted to do a transform on the data for some reason you want it there, but the regular transform function would be able to do that so it is redundant.